integration_tests/dbt_project.yml

```
@@ -1,7 +1,7 @@
 config-version: 2
 
 name: 'iterable_integration_tests'
-version: '0.2.0'
+version: '0.2.1'
```

Small fix to keep the two versions up to date for consistency
```suggestion
version: '0.3.1'
```